src/solver/variable/bindConstraints.hxx

```diff
@@ -102,25 +100,10 @@ inline void BindingConstraints<NextT>::provideInformations(I& infos)
     }
 }
 
-inline std::vector<uint> getInequalityBindConstraintGlobalNumbers(const Data::Study& study)
-{
-    std::vector<uint> bindConstGlobalNumbers;
-    Data::BindingConstraintRTI* allBindConst = study.runtime->bindingConstraint;
-    for (uint k = 0; k < study.runtime->bindingConstraintCount; k++)
-    {
-        // We pick only inequality binding constraints.
-        if (allBindConst[k].operatorType == '<' || allBindConst[k].operatorType == '>')
-        {
-            bindConstGlobalNumbers.push_back(k);
-        }
-    }
-    return bindConstGlobalNumbers;
-}
-
 template<class NextT>
 void BindingConstraints<NextT>::initializeFromStudy(Data::Study& study)
 {
-    std::vector<uint> InequalityBCnumbers = getInequalityBindConstraintGlobalNumbers(study);
+    const std::vector<uint>& InequalityBCnumbers = study.runtime->bindingConstraintGlobalNumbers;
```

Comment:
I'm not at ease with getting a data directly in the runtime class like this.
According to remarks made in source file **runtime.cpp**, we can either have : 
- **getInequalityBindConstraintGlobalNumbers**(study) computing the global numbers and return a vector of them (which it not costly with the vector move constructor)
-  **getInequalityBindConstraintGlobalNumbers**(study) could return a reference to the the global numbers as a data member of runtime class